nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -812,7 +832,7 @@ def compute_connectivity_output_optimized(self):
             ref_ip_blocks = self.config.get_referenced_ip_blocks(exclude_ipv6)
             for ip_block in ref_ip_blocks:
                 ip_blocks_mask |= ip_block
-            opt_peers_to_compare.filter_ipv6_blocks(ip_blocks_mask)
+            opt_peers_to_compare.filter_ip_blocks_by_mask(ip_blocks_mask)
```

Comment:
can't the arg to `filter_ip_blocks_by_mask` just be `IpBlock.get_all_ips_block(exclude_ipv6=True)` instead of `ip_blocks_mask` ? (and use the function `check_for_excluding_ipv6_addresses()` instead of using `self.output_config.excludeIPv6Range` )?

Reply:
Right, changed similarly to `TwoNetworkConfigsQuery.filter_conns_by_input_or_internal_constraints`

Reply:
opened issue #594 related to follow-ups for the ipv6 filtering aspects.